src/i18n.js

```diff
@@ -0,0 +1,619 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    // we init with resources
+    resources: {
+      en: {
+        translations: {
+          "Not Yet Obtained": "Not Yet Obtained",

```

Comment:
Here you can find detailed information about the resolution of those keys. 
https://www.i18next.com/principles/translation-resolution#resolution-order

So basically, yes if the key is not found it will use either a specified fallback key or the key itself.